src/alfasim_score/converter/alfacase/score_output_generator.py

```diff
@@ -31,30 +31,35 @@ def __init__(
         results_path: Path,
         well_start_position: Scalar,
         active_annuli: List[AnnulusLabel] = [],
+        walls: List[int] = [],
```

Comment:
@carlosnewmar Can't we use the built-in typings of `list` and `dict`?

Reply:
We can't here, unfortunelly, because it'll break older versions of Python during the building process.